se_feedback/sprint_7/feedback_sprint_7_all.md

```diff
@@ -0,0 +1,85 @@
+|      |            |
+|------|------------|
+|Group | BlueTurtle |
+|Assignment|Sprint 7|
+|Date|12/06/16|
+|TA|Bastiaan Reijm|
+
+#Sprint Feedback
+Feedback and Grades for Sprint 7.
+
+Total: **10**
+
+| User Story | Score |
+|------------|-------|
+| definition |   10   |
+| splitting  |   10   |
+| responsibility | 10  |
+
+| Learning from History | Score |
+|-----------------------|-------|
+| estimation            |  10    |
+| prioritisation        |  10    |
+| reflection            |  10   |
+
+## Notes
+* Good Job!
+* Looks like it was a smooth Sprint
+
+#Code Evolution Quality Feedback
+
+Total: **9.41**
+
+| Architecture                       | Score |
+|------------------------------------|-------|
+| Changes                            |   9   |
+| Architecture Design Document (ADD) |   8   |
+
+|                     | Score |
+|---------------------|-------|
+| Code Change Quality |   8    |
+
+| Code Readability | Score |
+|------------------|-------|
+| Formatting       |  10    |
+| Naming           |   9   |
+| Comments         |  10    |
+
+| Continuous Integration | Score |
+|------------------------|-------|
+| Building               |  10   |
+| Testing                |  10   |
+
+|         | Score |
+|---------|-------|
+| Tooling |  10   |
+
+| Pull-based Development | Score |
+|------------------------|-------|
+| Branching              |  10   |
+| Code Review            |  10   |
+
+##Notes
+* Parsers have high cyclomatic complexity (ex. PMDXMLParser)
```

Comment:
Hi, Bastiaan, I have already split the method in the parsers, and by cyclomatic complexity, do you mean they are too many independent paths? I have done following fix: #95 , did I misunderstand what you mean or is it also a fix as well?